### PR TITLE
Fix REPL run shortcut on macOS

### PR DESCRIPTION
**Pull request by fantix**
Refs edgedb/edgedb-cli#716

### CODE REVIEW

File: docs/guides/cheatsheet/repl.rst [hunk 1]
```diff
@@ -11,14 +11,20 @@ a semicolon and press "ENTER".
     db> select 5;
     {5}
 
-Alternatively, you can run the query without a semicolon with ⌥-Enter on macOS
+Alternatively, you can run the query without a semicolon with esc+return on macOS
 or Alt-Enter on Windows/Linux.
```
Comment (1st1) [inline docs/guides/cheatsheet/repl.rst:15]:
```suggestion
Alternatively, you can run the query without a semicolon by hitting an `ESC`
key immediately followed by `RETURN` on macOS, or Alt-Enter on Windows/Linux.
```

File: docs/guides/cheatsheet/repl.rst [hunk 2]
```diff
@@ -11,14 +11,20 @@ a semicolon and press "ENTER".
...
 .. code-block:: edgeql-repl
 
     db> select 5
     {5}
 
+.. tip::
+
+    This esc+return or Alt-Enter trick is also very useful when your cursor is in
+    the middle of a long query and you want to run it directly without moving the
+    cursor to the end of the query.
```
Comment (1st1) [inline docs/guides/cheatsheet/repl.rst:26]:
```suggestion
    The Alt-Enter / ESC+Return trick is also very useful when your cursor is in
    the middle of a long query and you want to run it directly without moving the
    cursor to the end of the query.
    
    Terminals like iTerm allow to configure left Option key to act as ESC+, which
    makes Alt-Enter also work on macOS.
```

File: docs/guides/cheatsheet/repl.rst (hunk 1 above)
Comment (colinhacks) [reply docs/guides/cheatsheet/repl.rst:15]: This works even if you hold down the `esc` key

File: docs/guides/cheatsheet/repl.rst (hunk 2 above)
Comment (colinhacks) [reply docs/guides/cheatsheet/repl.rst:26]: Let's mention the "Use Option as Meta option" here for Terminal.app users

Comment (colinhacks) [reply docs/guides/cheatsheet/repl.rst:26]: > Terminals like iTerm allow to configure

Edit: "Terminals like iTerm allow *you* to configure *the* left Option"

Comment (1st1) [reply docs/guides/cheatsheet/repl.rst]: Good catch and good suggestion on Terminal.app. I've updated with some detailed instructions for both iTerm and Terminal.app

File: docs/guides/cheatsheet/repl.rst [hunk 3]
```diff
@@ -11,14 +11,26 @@ a semicolon and press "ENTER".
...
 .. code-block:: edgeql-repl
 
     db> select 5
     {5}
 
+.. tip::
+
+    The Alt-Enter / ESC+Return trick is also very useful when your cursor is in
+    the middle of a long query and you want to run it directly without moving the
+    cursor to the end of the query.
```
Comment (1st1) [inline docs/guides/cheatsheet/repl.rst:26]:
```suggestion
    The Alt-Enter / ESC+Return trick is also very useful when your cursor is in
    the middle of a long query and you want to run it directly without moving 
    the cursor to the end of the query.
```

**Pull request merged**